packages/sources/css/src/components/tooltip/src/index.css

```diff
@@ -0,0 +1,143 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+@import '@vtmn/css-design-tokens/src/shadows';
+
+.vtmn-tooltip {
+  position: relative;
+}
+
+/* Applies to all tooltips */
+.vtmn-tooltip::before,
+.vtmn-tooltip::after {
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-3-font-size);
+  font-weight: var(--vtmn-typo_font-weight--regular);
+  line-height: var(--vtmn-typo_text-3-line-height);
+  box-shadow: var(--vtmn-shadow_200);
+  user-select: none;
+  pointer-events: none;
+  position: absolute;
+  display: none;
+  opacity: 1;
+  cursor: help;
+}
+.vtmn-tooltip::before {
+  content: '';
+  border: rem(8px) solid transparent;
+  z-index: 1000;
+}
+.vtmn-tooltip::after {
+  content: attr(data-tooltip);
+  width: fit-content;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding: rem(8px) rem(12px);
+  height: rem(36px);
+  border-radius: rem(4px);
+  background-color: var(--vtmn-color_black);
+  color: var(--vtmn-color_white);
+  z-index: 99;
+}
+
+/* Make the tooltips respond to hover */
+.vtmn-tooltip:hover::before,
+.vtmn-tooltip:hover::after,
+.vtmn-tooltip:focus-visible::before,
+.vtmn-tooltip:focus-visible::after {
+  display: flex;
```

Comment:
What do you think about adding a `cursor: help`?

Reply:
I am not fan of this cursor, it doesn't bring added value to the tooltip.

Reply:
But can discuss that with the designers

Reply:
Not neccessary but cursor pointer added to show clicable element